blueprints/validator/files/tests/unit/validators/__name__-test.js

```diff
@@ -1,9 +1,8 @@
 import Ember from 'ember';
-import { moduleFor, test }
-from 'ember-qunit';
+import { moduleFor, test } from 'ember-qunit';
 
 moduleFor('validator:<%= dasherizedModuleName %>', 'Unit | Validator | <%= dasherizedModuleName %>', {
-  needs: ['validator:messages']
+  integration: true
```

Comment:
We ran into some registry/container issues with ember-release and above. Converting if to an integration test helped create the container we needed for the test. If you believe this should be a unit test, I can keep as it was. Might just need some more testing though.